src/models/tiles/geometry/geometry-model.ts

```diff
@@ -250,6 +250,10 @@ export const PolygonModel = GeometryObjectModel
   .props({
     type: typeField("polygon"),
     points: types.array(types.string),
+    labelOption: types.optional(
+      types.enumeration<ELabelOption>("LabelOption", Object.values(ELabelOption)),
+      ELabelOption.kNone),
```

Comment:
I'd be inclined to use:
```suggestion
    labelOption: types.maybe(
      types.enumeration<ELabelOption>("LabelOption", Object.values(ELabelOption))),
```
and let `undefined` be treated as none, but I don't feel strongly about it. Then you wouldn't have to change as many tests, for instance.

Reply:
Yeah, I can see that would have been easier. But since the work of updating the tests is already done, I'm going to leave this.